src/linechart/LineChart.jsx

```diff
@@ -55,86 +46,71 @@ module.exports = React.createClass({
     innerWidth = this.getOuterDimensions().width - props.margins.left - props.margins.right;
     innerHeight = this.getOuterDimensions().height - props.margins.top - props.margins.bottom;
 
-    if (!Array.isArray(data)) {
-      data = [data];
+    if (!Array.isArray(props.data)) {
```

Comment:
This isn't needed anymore since `props.data` cannot be anything except an array based on the changes I made in #227